### ISSUE TITLE
Profile Photo Preview in Account Details displaying broken image

### ISSUE THREAD

**Issue by cnorm35** [bullet-train-co/bullet_train-core#464]
### Steps to Reproduce
With a new account, uploading a profile photo from the Account Details section displays a broken image

### Additional Details
The image (png) is uploaded and is displaying the correct image in the Account Avatar within the top menu.

I saw this in the Rails logs and have ran into this issue before and believe it's related to vips image processing

```
web              |   Disk Storage (0.2ms) Downloaded file from key: vphget8zh1qi8wip2txs7jq7qahl
web              | objc[65257]: +[__NSCFConstantString initialize] may have been in progress in another thread when fork() was called.
web              | objc[65257]: +[__NSCFConstantString initialize] may have been in progress in another thread when fork() was called. We cannot safely call it or ignore it in the fork() child process. Crashing instead. Set a breakpoint on objc_initializeAfterForkError to debug.
web              | [64483] - Worker 1 (PID: 65263) booted in 0.0s, phase: 0
```

Commenting out `# workers ENV.fetch("WEB_CONCURRENCY", 4)` from `config/puma.rb` and restarting the rails server resolves this issue.  

### Screenshots

Displaying broken Image
![Screenshot 2023-08-23 at 11 51 43 AM](https://github.com/bullet-train-co/bullet_train-core/assets/4549543/3ebb0f04-5d2a-4d4c-ae2d-08d582ee8eca)

Displaying correct image after updating Puma config
![Screenshot 2023-08-23 at 11 52 25 AM](https://github.com/bullet-train-co/bullet_train-core/assets/4549543/4f75a0ae-affe-4bd1-a23a-aa210c9be63f)

**Comment by jagthedrummer** [comment]
I just looked into this and the fix of disabling puma workers didn't do anything for me. After poking around I think we're just not trying to render an image.

@gazayas, it looks like the not rendering an image is related to the recent addition of adding ActiveStorage as an upload option.

https://github.com/bullet-train-co/bullet_train-core/blob/2b9dea3798d625943d12c1a959d817cac71122df/bullet_train/app/views/account/users/_form.html.erb#L17-L21

I'll open an issue about that and link it here.

**Comment by cnorm35** [comment]
@jagthedrummer Thanks for looking into that! That issue may have just been something I was running into locally.  I've seen the 

`objc[65257]: +[__NSCFConstantString initialize] may have been in progress in another thread when fork() was called. We cannot safely call it or ignore it in the fork() child process. Crashing instead. Set a breakpoint on objc_initializeAfterForkError to debug` 

error before on another project and believe it was related to vips image processing on OSX and running specs parallel.

Let me know if you'd like me to close this issue.

**Comment by jagthedrummer** [comment]
After getting to where we try to render an image (WIP PR #515) I was able to recreate this issue:

```
15:06:28 web.1              | Processing by ActiveStorage::Representations::RedirectController#show as PNG
15:06:28 web.1              |   Parameters: {"signed_blob_id"=>"eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBDQT09IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--f19f613de72e1c4e2b0a23fd1f8bcdf93fd23519", "variation_key"=>"[FILTERED]", "filename"=>"Do you want avocados or not"}
15:06:28 web.1              |   ActiveStorage::Blob Load (0.8ms)  SELECT "active_storage_blobs".* FROM "active_storage_blobs" WHERE "active_storage_blobs"."id" = $1 LIMIT $2  [["id", 3], ["LIMIT", 1]]
15:06:28 web.1              |   ActiveStorage::VariantRecord Load (1.1ms)  SELECT "active_storage_variant_records".* FROM "active_storage_variant_records" WHERE "active_storage_variant_records"."blob_id" = $1 AND "active_storage_variant_records"."variation_digest" = $2 LIMIT $3  [["blob_id", 3], ["variation_digest", "uyx6Kcit1Aa78Mrn7bVgZ7OZn0Y="], ["LIMIT", 1]]
15:06:28 web.1              |   Disk Storage (0.4ms) Downloaded file from key: zwm14lvqm1m0q6n0f5zsauww4ur5
15:06:28 web.1              | objc[18525]: +[__NSCFConstantString initialize] may have been in progress in another thread when fork() was called.
15:06:28 web.1              | objc[18525]: +[__NSCFConstantString initialize] may have been in progress in another thread when fork() was called. We cannot safely call it or ignore it in the fork() child process. Crashing instead. Set a breakpoint on objc_initializeAfterForkError to debug.
15:06:28 web.1              | [17939] - Worker 1 (PID: 18531) booted in 0.0s, phase: 0
```

![CleanShot 2023-09-01 at 15 07 55](https://github.com/bullet-train-co/bullet_train-core/assets/58702/b0cd0545-3e36-42ca-b065-3c9da7b1f7af)

Commenting out this line, or setting the value to `0` gets it to work.

https://github.com/bullet-train-co/bullet_train/blob/122f6953f9eb5df3d670923cda38511a45d4772a/config/puma.rb#L33

**Comment by cnorm35** [comment]
Not sure if it's helpful, but I've seen those same warning pop up in system tests that were logging that error, passing `OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES` seems to resolve it

**Comment by jagthedrummer** [comment]
@cnorm35 after poking around a bit I found a way to solve this without disabling puma workers.

https://github.com/libvips/ruby-vips/issues/155#issuecomment-1047370993

You can add `gem "ruby-vips"` in the `:development, :test` group of the `Gemfile` did the trick for me.

I'll whip up a PR for that real quick.

**Comment by jagthedrummer** [comment]
PR to fix this:

https://github.com/bullet-train-co/bullet_train/pull/959/files

**Comment by cnorm35** [comment]
Awesome, thanks so much for the update!

**Comment by jagthedrummer** [comment]
I'm going to leave this open until we have a fix for #498, just so we can be really really sure we got it.

**Comment by jagthedrummer** [comment]
OK, this should be fixed with the merge of #515. I'll be cutting a new release for version 1.4.5 shortly.
